addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -84,16 +85,19 @@ public void run() {
                             VIRTUAL_DATAPOINT_NAME_FIRMWARE);
                     setProperty(properties, channelZero, Thing.PROPERTY_SERIAL_NUMBER, device.getAddress());
                     setProperty(properties, channelZero, PROPERTY_AES_KEY, DATAPOINT_NAME_AES_KEY);
+                    properties.put(PROPERTY_ADDRESS, device.getAddress());
```

Comment:
I thought the "address" is a configuration parameter? Why do you add it as a property?